components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/core/Constants.java

```diff
@@ -49,7 +53,9 @@
                         "user: %s"),
         ERROR_CODE_GET_APP_BY_USER("15005", "Error retrieving authorized applications",
                 "A system error occurred while retrieving authorized applications for " +
-                        "user: %s");
+                        "user: %s"),
+        ERROR_CODE_REVOKE_TOKEN_BY_APP_ID("15006", "Error revoking issued tokens", "A system " +
```

Comment:
`"Error revoking tokens","An error occurred while revoking tokens for application: %s in tenant domain: %s"` WDYT?